indico/htdocs/sass/base/_theme.scss

```diff
@@ -14,3 +14,67 @@
  * You should have received a copy of the GNU General Public License
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
+
+%semantic-background {
+    color: white;
+    background: $dark-gray;
+
+    &.highlight {
+        background: $blue;
+    }
+
+    &.success {
+        background: $green;
+    }
+
+    &.error {
+        background: $red;
+    }
+
+    &.warning {
+        background: $yellow;
+    }
+
+    &.visited {
```

Comment:
Is that really what we had decided on last week? I thought we'd picked something else...